bundle-size/test/api.test.js

```diff
@@ -418,153 +446,36 @@ describe('bundle-size api', () => {
-        const nocks = await lastNetworkRequest;
-        nocks.done();
-      }
-    );
-
-    test(
-      'update check on bundle-size report (report/_delayed_-base = ' +
-        '12.34KB/12.23KB)',
-      async () => {
-        await db('checks').insert({
-          head_sha: '26ddec3fbbd3c7bd94e05a701c8b8c3ea8826faa',
-          owner: 'ampproject',
-          repo: 'amphtml',
-          pull_request_id: 19603,
-          installation_id: 123456,
-          check_run_id: 555555,
-          delta: null,
-          approving_teams: null,
-        });
-
-        const baseBundleSizeFixture = getFixture(
-          '5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
-        );
-        baseBundleSizeFixture.content = Buffer.from(
-          '{"dist/v0.js":12.23}'
-        ).toString('base64');
-        const lastNetworkRequest = new Promise(resolve => {
-          const nocks = nock('https://api.github.com')
-            .get(
-              '/repos/ampproject/amphtml-build-artifacts/contents/' +
-                'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
-            )
-            .times(2)
-            .reply(404)
-            .get(
-              '/repos/ampproject/amphtml-build-artifacts/contents/' +
-                'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
-            )
-            .reply(200, baseBundleSizeFixture)
-            .patch('/repos/ampproject/amphtml/check-runs/555555', body => {
+          .reply(200, getFixture('reviews'))
+          .post(
+            '/repos/ampproject/amphtml/pulls/19603/requested_reviewers',
+            body => {
               expect(body).toMatchObject({
-                conclusion: 'action_required',
-                output: {
-                  title: 'Δ +0.11KB | approval required',
-                },
+                reviewers: [
+                  expect.stringMatching(
+                    /kristoferbaxter|jridgewell|alanorozco|erwinmombay|kevinkimball|choumx/
+                  ),
+                ],
               });
               return true;
-            })
-            .reply(200)
-            .get('/repos/ampproject/amphtml/pulls/19603/requested_reviewers')
-            .reply(200, getFixture('requested_reviewers'))
-            .get('/repos/ampproject/amphtml/pulls/19603/reviews')
-            .reply(200, getFixture('reviews'))
-            .post(
-              '/repos/ampproject/amphtml/pulls/19603/requested_reviewers',
-              body => {
-                expect(body).toMatchObject({
-                  reviewers: [expect.stringMatching(/erwinmombay|estherkim/)],
-                });
-                return true;
-              }
-            )
-            .reply(200, () => {
-              setTimeout(() => {
-                resolve(nocks);
-              }, 0);
-            });
-        });
+            }
+          )
+          .reply(200, () => {
+            setTimeout(() => {
+              resolve(nocks);
+            }, 0);
```

Comment:
`setTimeout(cb)` defaults to a 0 timeout; the 0 isn't required.

Reply:
TIL!